mcs/tools/corcompare/Util.cs

```diff
@@ -64,7 +75,23 @@ internal TypeDefinition GetBaseType (TypeDefinition child)
 			if (child.BaseType == null)
 				return null;
 
-			return child.BaseType.Resolve ();
+			try {
+				return child.BaseType.Resolve ();
+			} catch (AssemblyResolutionException) when (IgnoreResolutionErrors) {
+				return null;
```

Comment:
If we can't find the base type, just continue. If we really cared, then we should have added the dependency to the resolver. If we want this to fail, then don't set the flag to true.